packages/app/src/app/pages/Dashboard/Content/routes/Recent/index.tsx

```diff
@@ -2,28 +2,43 @@ import React, { useEffect } from 'react';
 import { useAppState, useActions } from 'app/overmind';
 import { sandboxesTypes } from 'app/overmind/namespaces/dashboard/types';
 import { Header } from 'app/pages/Dashboard/Components/Header';
-import { VariableGrid } from 'app/pages/Dashboard/Components/VariableGrid';
+import {
+  GRID_MAX_WIDTH,
+  GUTTER,
+  VariableGrid,
+} from 'app/pages/Dashboard/Components/VariableGrid';
 import { SelectionProvider } from 'app/pages/Dashboard/Components/Selection';
 import { Helmet } from 'react-helmet';
 import { DashboardGridItem, PageTypes } from 'app/pages/Dashboard/types';
+import { Element } from '@codesandbox/components';
+import { UpgradeBanner } from 'app/pages/Dashboard/Components/UpgradeBanner';
+import { SubscriptionType, TeamMemberAuthorization } from 'app/graphql/types';
 
 export const Recent = () => {
   const {
-    activeTeam,
+    activeTeamInfo,
+    activeWorkspaceAuthorization,
     dashboard: { sandboxes },
   } = useAppState();
   const {
     dashboard: { getPage },
   } = useActions();
 
+  const activeTeamId = activeTeamInfo?.id;
+
   useEffect(() => {
     getPage(sandboxesTypes.RECENT);
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [activeTeam]);
+  }, [activeTeamId]);
 
   const dataIsLoading =
     sandboxes.RECENT_BRANCHES === null || sandboxes.RECENT_SANDBOXES === null;
 
+  const isPro = [
+    SubscriptionType.PersonalPro,
+    SubscriptionType.TeamPro,
+  ].includes(activeTeamInfo?.subscription?.type);
```

Comment:
again pointing to the boolean logic implemented in the sidebar, I think a better test is the `subscription.status` which should be either `ACTIVE` or `TRIALING`, otherwise the user is not actually on PRO